Jenkinsfile

```diff
@@ -30,49 +31,94 @@ pipeline {
     }
 
     stages {
-
-        stage('Build the PR commit image') {
-            steps {
-                withVault([configuration: configuration, vaultSecrets: secrets]) {
-                    sh 'bash -x build_deploy.sh'
-                }
-            }
-        }
-
         stage('Run Tests') {
             parallel {
-                stage('Run unit tests') {
+                stage('Build the PR commit image') {
                     steps {
                         withVault([configuration: configuration, vaultSecrets: secrets]) {
-                            sh 'bash -x ./scripts/unit_test.sh'
+                            sh '''
+                            env
+                            
+                            IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+                            
+                            bash -x build_deploy.sh
+                            '''
                         }
                     }
                 }
-                stage('Run smoke tests') {
+
+                // stage('Run unit tests') {
+                //     steps {
+                //         withVault([configuration: configuration, vaultSecrets: secrets]) {
+                //             sh 'bash -x ./scripts/unit_test.sh'
+                //         }
+                //     }
+                // }
+
+                stage('Deploy Ephemeral Environment & Components') {
+                    environment {
+                        RELEASE_NAMESPACE="false"
+                    }
                     steps {
                         withVault([configuration: configuration, vaultSecrets: secrets]) {
                             sh '''
                                 curl -s ${CICD_URL}/bootstrap.sh > .cicd_bootstrap.sh
                                 source ./.cicd_bootstrap.sh
+
+                                PREVIOUS_COMMIT_HASH=$(git rev-parse --short=7 HEAD^)
+                                GIT_COMMIT=$(git rev-parse HEAD^)
+                                IMAGE_TAG="pr-${ghprbPullId}-${PREVIOUS_COMMIT_HASH}"
+
                                 source "${CICD_ROOT}/deploy_ephemeral_env.sh"
-                                source "${CICD_ROOT}/cji_smoke_test.sh"
+
+                                > reserved_namespace
+                                echo "$NAMESPACE" >> reserved_namespace
                             '''
                         }
-                    }
-                    post {
-                        failure {
-                            slackSend  channel: '@eshamard', color: "danger", message: "Smoke tests failed in Compliance PR check. <${env.ghprbPullLink}|PR link>  (<${env.BUILD_URL}|Build>)"
-                        }
-                        unstable {
-                            slackSend  channel: '@eshamard', color: "warning", message: "Smoke tests failed in Compliance PR check. <${env.ghprbPullLink}|PR link>  (<${env.BUILD_URL}|Build>)"
```

Comment:
are these notifications really meant to be removed as part of this PR?  They seem to notify a specific QE if the smoke tests failed.

Reply:
OH whoops- no they shouldn't be removed.